### ISSUE TITLE
Move the Code portion of the landing page outside

### ISSUE THREAD

**Issue by ahuang11** [holoviz/panel#6692]
Was reading:
https://python.plainenglish.io/top-5-python-frontend-libraries-for-data-science-part-2-4d07a48d2fde

And they chose the example from [build dashboard](https://panel.holoviz.org/tutorials/basic/build_dashboard.html) and concluded "it has a steeper learning curve, and it takes more time to grasp its core concepts" which is not even an apples to apples comparison.

I think we should make it clear to the user that the code is much more straightforward, exposing the code outside of the tabs:

<img width="835" alt="image" src="https://github.com/holoviz/panel/assets/15331990/48d7bb38-e23d-45b0-abb4-2b315c263d2d">

from https://panel.holoviz.org/index.html

Or even a simpler example mirroring the gradio one:
```python
import gradio as gr

def greet(name, intensity):
    return "Hello, " + name + "!" * int(intensity)

demo = gr.Interface(
    fn=greet,
    inputs=["text", "slider"],
    outputs=["text"],
)

demo.launch()
```

**Comment by philippjfr** [comment]
Yes, I don't think showing a minimal example would have resulted in them picking that example for their comparison. The example on the homepage at this point is a little outdated in any case and it'd be nice to produce a polished video instead.

**Comment by ahuang11** [comment]
Do you have a code snippet in mind to do a video demo of, preferably one that builds up to the full app? I'm happy to record if we agree on one.
